tests/Gedmo/Loggable/Fixture/Entity/Address.php

```diff
@@ -16,10 +16,10 @@
 class Address
 {
     /**
-     * @var string
+     * @var int
      * @ORM\Id()
-     * @ORM\Column(type="string", length=36)
-     * @ORM\GeneratedValue(strategy="UUID")
```

Comment:
The UUID generator is removed in DBAL 3, and support for it is deprecated in ORM 2.10.  If the intent is to have a test with a UUID PK, then the test environment needs to be updated to handle UUID generation.  For now, I switched to a traditional integer PK to avoid breaking things with DBAL 3 installed.